spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientHttpRequestFactory.java

```diff
@@ -38,12 +34,8 @@
 
 	private final SpringClientFactory clientFactory;
 
-	private LoadBalancerClient loadBalancer;
```

Comment:
Yeah. We plug into the `RestClient` here https://github.com/spring-cloud/spring-cloud-netflix/blob/master/spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfiguration.java#L170-L177